### PR TITLE
Multiple board

### PR DESCRIPTION
**Pull request by milkcanbass**
Updated path of FE and logic for redirect to implement multiple boards.
Since this change affects many sections, please review it carefully.
Especially, Sign in and Sign up logic need review.
I spent more time to debug than usual,  but there might be some issues.

Here are the major changes on this pr
-User can select dashboard by clicking three dots on the blue nav
-Added dashboard Id to local storage to save the latest dashboard user accessed. Because of this, the user can access the board he used before he logout.
-Add a drop-down menu to the user icon on the top bar. he can logout from here.
-Displayed Add column button if there is no column on the board

### CODE REVIEW

File: client/src/utils/handleUpdateTasks.js [hunk 1]
```diff
@@ -75,7 +70,35 @@ export const addDashboard = (title, cb) => {
...
-      console.log(err);
+      handleError(err);
+    });
+};
+
+export const getDashboard = (dashboardId, cb) => {
+  let body = {
+    dashboardId
+  };
+
```
Comment (zeinabfarag-zz) [inline client/src/utils/handleUpdateTasks.js]: why are you making a post request to get dashboard? also dashboard Id is in params so no need to add it to the body variable you have. I would change this so you have a route to get dashboard on the backend as well as a route to post to dashboard

**Review by zeinabfarag-zz — CHANGES_REQUESTED** [review]

**Comment by zeinabfarag-zz** [comment]
I would also remove the dashboard has been loaded success alert cuz its a little distracting.  I will review this in more detail once I make my calendar pr and these changes have been made. @milkcanbass

**Comment by zeinabfarag-zz** [comment]
in terms of ui i would use https://material-ui.com/components/drawers/ to display the boards so once the use clicks on those three dots on the blue nav, the drawer shows up on the right with the dashboards.

File: client/src/utils/handleUpdateTasks.js (hunk 1 above)
Comment (milkcanbass) [reply client/src/utils/handleUpdateTasks.js]: thanks. fixed.

**Comment by milkcanbass** [comment]
I'll try to use use https://material-ui.com/components/drawers/ . Since current menu is functional, this task is low priority. I'll create other branch for this task.

File: client/src/utils/handleUpdateTasks.js (hunk 1 above)
Comment (milkcanbass) [reply client/src/utils/handleUpdateTasks.js]: In terms of drawer. I'll create other branch for the task.

**Review by milkcanbass — COMMENTED** [review]
changed

**Pull request merged**